lib/tasks/db_data_migrate.rake

```diff
@@ -68,24 +81,23 @@ def fill_new_authentication_service_attributes
 end
 
 def create_missing_fingerprints
-  #uploads = Upload.where.not(fingerprint_value: nil)
-  uploads = Upload.eager_load(:fingerprints).where('fingerprints.id is NULL').where.not(fingerprint_value: nil, completed_at: nil).unscope(:order)
   fingerprint_count = Fingerprint.count
   failures = []
-  puts "Creating fingerprints for #{uploads.count} uploads"
```

Comment:
I don't think that you need to get rid of the `uploads` variable and the line that prints out the `uploads.count`. The variable stores an ActiveRecord relation that is lazy about making the call to retrieve objects. So, you should be able to just call `uploads.find_in_batches`. Of course, it doesn't really matter, since there shouldn't be any Uploads at this point that are missing Fingerprints.